examples/tv-casting-app/linux/main.cpp

```diff
@@ -109,6 +109,10 @@ int main(int argc, char * argv[])
 {
     VerifyOrDie(CHIP_NO_ERROR == chip::Platform::MemoryInit());
     VerifyOrDie(CHIP_NO_ERROR == chip::DeviceLayer::PlatformMgr().InitChipStack());
+    // Enter commissioning mode, open commissioning window
```

Comment:
does this really work? Seems like initialization of KeyValueStoreMgr needs to happen before the server gets inited

Reply:
I think you need to move this down to just before the call to:

SuccessOrExit(err = CastingServer::GetInstance()->DiscoverCommissioners());

Reply:
Moved it. Thanks!